README.md

```
@@ -20,6 +20,32 @@ If you have any questions, please join our [Discord](https://discord.gg/zaGrfqFE
 If you enjoy my open source work and would like to support me, consider buying me a coffee! Your support helps me stay caffeinated and motivated to keep improving and creating awesome projects.
 
+## Quest Helper Integration
+
+The integrated Quest Helper works together with other plugins to give a nice unified experience while questing.
+
+### Not Enough Runes
+
+If you have the [Not Enough Runes](https://runelite.net/plugin-hub/show/not-enough-runes) plugin installed, you can
+right-click item requirements and click `Go to NER...` to look that item up in Not Enough Runes.
+
+![](./images/not-enough-runes-01.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)